tasks/models.py

```diff
@@ -1,3 +1,39 @@
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+
+class Tag(models.Model):
+    tag_name = models.CharField(max_length=50, unique=True, primary_key=True)
+
+    def __str__(self):
+        return self.tag_name
+
+
+class Task(models.Model):
+    task_code = models.CharField(max_length=8, unique=True, primary_key=True)
+    title = models.CharField(max_length=255)
+    author = models.ForeignKey(User, on_delete=models.CASCADE, to_field="username")
```

Comment:
models.CASCADE is probably a bad idea since the author is not important anyway, so we can use models.SET_NULL to set the field to NULL when the referenced author is deleted (if it's models.CASCADE then the task is also deleted)